lib/schema/pumpSettings.js

```diff
@@ -100,7 +100,9 @@ module.exports = schema.makeHandler('pumpSettings', {
     insulinSensitivity: schema.ifExists(insulinSensitivitySchema),
     insulinSensitivities: schema.ifExists(schema.isObjectWithValueSchema(insulinSensitivitySchema)),
     bgTarget: schema.ifExists(bgTargetArraySchema),
-    bgTargets: schema.ifExists(schema.isObjectWithValueSchema(bgTargetArraySchema))
+    bgTargets: schema.ifExists(schema.isObjectWithValueSchema(bgTargetArraySchema)),
+    automatedDelivery: schema.ifExists(schema.in('true', 'false')),
```

Comment:
Is this a boolean? `true` and `false` as opposed to a string `"true"` and `"false"`?

Reply:
Ah, good catch. It still works when these are strings, but that's maybe more about how JS's concept of truthiness, so I'll change these to actual booleans.